webpack.config.js

```diff
@@ -136,14 +138,22 @@ const common = merge([
         THREE: 'three',
         jQuery: 'jquery',
       }),
+      new HtmlWebpackPlugin({
+        title: 'Nuxeo',
+        template: 'index.html',
+        nuxeo: {
+          packages: JSON.stringify((process.env.NUXEO_PACKAGES || '').split(' ').filter((_) => !!_)),
```

Comment:
Perhaps it would be better to split this by commas, so we don't have to specify the list wrapped in quotes?

Reply:
comma sounds more natural and consistent to me too

Reply:
After checking with Nelson, it seems that the space-separated list was used here to be compatible with what's being already set for the docker images, so we could re-use it. I'd suggest splitting by both space and comma, so it could serve both purposes, something like:
```suggestion
          packages: JSON.stringify((process.env.NUXEO_PACKAGES || '').split(/[\s,]+/).filter(Boolean)),
```

Reply:
I agree with you. I kept spaces for compat with `NUXEO_PACKAGES` used by our official Docker image so you can just reuse the same env var for both server and ui.

Reply:
So, no splitting by both spaces and commas?